nimble/host/services/auracast/syscfg.yml

```diff
@@ -0,0 +1,29 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+  BLE_SVC_AURACAST_ADV_INSTANCE:
+    description: >
+      Defines which advertising instance is used for Auracast.
+      If set to -1 last advertising instance is used, and if MESH is enabled, one before that.
```

Comment:
What is the reason for defining in configuring which advertising instance should be used ?

Reply:
For broadcast, we can choose. For Auracast user has freedom to define single instance (for now) to be used, OR give up this decision to package. This can simplify process of writting apps, especially simple ones